build.gradle

```diff
@@ -89,7 +89,7 @@ jenkinsPlugin {
 configurations { forceGroovy }
 
 dependencies {
-    forceGroovy 'org.codehaus.groovy:groovy-all:2.4.8'
+    forceGroovy "org.codehaus.groovy:groovy-all:${groovyVersion}"
```

Comment:
The integration test also requires an update - see https://github.com/jenkinsci/groovy-events-listener-plugin/blob/1.016/src/test/groovy/org/jenkinsci/plugins/globalEventsPlugin/integration/IntegrationTest.groovy#L29.